src/kafkaaggregator/testtopic/agents.py

```diff
@@ -72,7 +73,7 @@ def aggregate(timestamp: float, messages: List[TestTopic]) -> AggTestTopic:
     return aggregated_message
 
 
-def process_window(key: Tuple, value: List[TestTopic]) -> None:
+def process_window(key: Tuple, value: List) -> None:
```

Comment:
Are you running mypy? I'm surprised it doesn't need you to suggest the type of the items.

Reply:
Yep, mypy does not complain about the type of the items, for this argument `List[Any]` seems appropriate.